packages/matchbox/src/components/Button/Button.js

```diff
@@ -1,140 +1,146 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
+import { deprecate } from '../../helpers/propTypes';
+import styled from 'styled-components';
+import { Box } from '../Box';
 
 import Group from './Group';
-import styles from './Button.module.scss';
-
-class Button extends Component {
-  static displayName = 'Button';
-
-  static Group = Group;
-
-  static propTypes = {
-    color: PropTypes.oneOf(['orange', 'blue', 'navy', 'purple', 'red']),
-    disabled: PropTypes.bool,
-    destructive: PropTypes.bool,
-    flat: PropTypes.bool,
-    outline: PropTypes.bool,
-    size: PropTypes.oneOf(['small', 'large', 'default']),
-    fullWidth: PropTypes.bool,
-    submit: PropTypes.bool,
-    to: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-    external: PropTypes.bool,
-    component: PropTypes.oneOfType([
-      PropTypes.func,
-      PropTypes.element
-    ]),
-    children: PropTypes.node
-  }
-
-  static defaultProps = {
-    size: 'default'
-  }
+import { base, size, colorVariant, disabled, fullWidth, group } from './styles';
+
+const StyledButton = styled(Box)`
+  ${base}
+  ${size}
```

Comment:
This should probably be renamed to avoid conflicts with `styled-system` in the future